defaults/main.yml

```diff
@@ -166,6 +166,10 @@ session_management_cors_origin:
 session_management_oauth_client_id:
 session_management_oauth_client_secret:
 session_management_ip_stack_api_token: "{{ cloudxr_configuration.ip_stack_api_token }}"
+session_management_adaptive_instance_scaling_enabled: 'true'
```

Comment:
to be consistent throughout this role, we should actually use boolean values for boolean values and then convert them to strings when needed (e.g. if used as an environment variable).

```suggestion
session_management_adaptive_instance_scaling_enabled: true
```

Reply:
I tried that already but it creates this error:
![image](https://user-images.githubusercontent.com/10352866/143858659-fcb4c760-d851-4e92-9c3a-ddf749017e57.png)

Reply:
ah, in combination with ternary it makes sense.